app/helpers.php

```diff
@@ -209,20 +209,18 @@ function lazy_load_image($url, $class = '', $alt = '')
 
 function nav_links()
 {
-    $links = [];
+    $links = landing_nav_links();
```

Comment:
tbh the order *should* be the same in both instances.. with some hidden but yeah.